Sources/Commands/SwiftBuildTool.swift

```diff
@@ -213,14 +211,14 @@ public struct SwiftBuildTool {
         print("USAGE: swift build [mode] [options]")
         print("")
         print("MODES:")
-        print("  --configuration <value>        Build with configuration (debug|release) [-c]")
-        print("  --clean[=<mode>]               Delete artifacts (build|dist)")
+        print("  --configuration <value>   Build with configuration (debug|release)")
+        print("  --clean [<mode>]          Delete artifacts (build|dist)")
```

Comment:
This makes it look like the mode is optional, but is it actually still optional? This means it won't parse properly except at the end.